Libplanet/HashDigest.cs

```diff
@@ -243,6 +254,11 @@ bool IEquatable<HashDigest<T>>.Equals(HashDigest<T> other)
 
             return true;
         }
+
+        public void GetObjectData(SerializationInfo info, StreamingContext context)
+        {
+            info.AddValue("hashDigest", ToByteArray());
```

Comment:
I guess it would be safer against mistakes(e.g. typo) and refactor. 😅 

```suggestion
            info.AddValue(nameof(HashDigest<T>), ToByteArray());
```